.travis.yml

```diff
@@ -1,7 +1,23 @@
 language: php
 
 php:
-    - 5.3
-    - 5.4
-    - 5.5
-    - 5.6
+  - 5.3
+  - 5.4
+  - 5.5
+  - 5.6
```

Comment:
[5.5 or earlier versions have reached EoL](http://php.net/supported-versions.php). those versions should be removed from supported versions of this package, I think.